Test/ParallelPrimitives/RadixSort.cpp

```diff
@@ -120,14 +122,44 @@ void RadixSort::compileKernels( oroDevice device )
 	if( m_flags & FLAG_LOG ) RadixSortImpl::printKernelInfo( oroFunctions[Kernel::SORT_SINGLE_PASS] );
 }
 
-void RadixSort::configure( oroDevice device, u32& tempBufferSizeOut )
+int RadixSort::calculateWGsToExecute( oroDevice device ) noexcept
 {
-	oroDeviceProp props;
+	oroDeviceProp props{};
 	oroGetDeviceProperties( &props, device );
-	const int occupancy = 8; // todo. change me
 
-	const auto newWGsToExecute{ props.multiProcessorCount * occupancy };
+	constexpr auto maxWGSize = std::max( { COUNT_WG_SIZE, SCAN_WG_SIZE, SORT_WG_SIZE } );
+	const int warpSize = ( props.warpSize != 0 ) ? props.warpSize : 32;
```

Comment:
This is a workaround for Nvidia GPU since the API returns 0 for `props.warpSize` . This should be solved by #16